\section{Aim 3. Assess the impact of PRC2 dysregulation on the tissue-specificity of cancer driver genes.}

\subsection*{Rationale}

As explained by Haigis \emph{et al.} in \cite{Haigis2019}, substantial evidence has lead us to hypothesize that the potential for a gene to drive cancer is predominately determined by the preexisting state of the cell and tissue microenvironment.
This state is determined by the epigenetic structure established over the development of the tissue.
By regulating gene expression, the epigenetic state establishes the transcriptome and proteome, defining the cellular signaling network.
Only the disruption to specific genes and proteins are oncogenic within the context of this network, thus defining the tissue-specificity of tumor driver genes.

PRC2 is responsible for regulating gene expression by methylating lysine 27 of histone H3 (H3K27), and its activity is often ascribed to maintaining a cell's functional identity \cite{Comet2016MaintainingCancer., Laugesen2019a}.
Because the genes comprising this protein complex are frequently mutated in cancer, we propose it as a system through which to investigate the impact of the epigenome on tissue-specificity of cancer drivers.
For this study, we propose using COAD and LUAD as they both have high frequency of gain-of-function alterations to PRC2 through amplification or over-expression of \emph{EZH2}, and our lab and collaborators have extensive experience with these two diseases.
In addition, we have been advised by Kristian Helin to use skin melanoma because...
\textbf{We hypothesize that the dysregulation of PRC2 methylation alters the state of a cell, and consequently alters the oncogenic potential of cancer driver genes.}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Aim 1.1
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection*{Aim 1.1. Identify recurring concomitant mutations in PRC2-mutant tumors.}


\subsubsection*{Approach}

If the epigenetic state of a cell determines its susceptibility to different cancer driver genes, then we expect to find different genes mutated when PRC2 function is altered.
To this end, we will identify genes that are more often mutated in PRC2-mutant tumors compared to those with normal PRC2. 
Of these genes, the properties of the known oncogenes, along with a functional enrichment analysis of the others, will describe the distinct changes induced by PRC2-mediated epigenetic changes.

In addition, we can identify more complex rearrangements of the cellular signaling structure by identifying new modules of comutating or mutually-exclusive genes in PRC2-mutant tumors \cite{Miller2011, Vandin2012, Ciriello2012, Jia2014, Zhang2014c, Ahmed2015, Kim2015, Leiserson2015, Babur2015, Leiserson2015b, Dao2017, Leiserson2016, Cho2016a, Reyna2018, Zhang2018e, Bokhari2020QuaDMutNetEx:Frequency.}.
Genes often comutate because the events cooperate in driving cancer or are mutually-exclusive because they are redundant, synthetic lethal \cite{Kaelin2005}, or collateral lethal \cite{Muller2015}.
Therefore, comutation or mutually exclusive interactions that are unique or lost in PRC2-mutant tumors suggests that the epigenetic alterations have greatly influenced the signaling pathways.

It is known that chromatin structure plays a substantial role in the distribution of mutations along the chromosomes \cite{Schuster-Bockler2012, Polak2015, Gonzalez-Perez2019}.
Thus, the possibility of the comutation events described earlier could be increased by changes in their proximity in 3D-space due to altered PRC2 behaviour.
We will include this in the analysis by identifying "spatial comutation hotspots" \cite{Shi2016ChromatinGenes}, thus linking dysregulated PRC2-mediated chromatin organization to the increased rates of mutation or changes in comutation/mutually exclusive modules.

Overall, the analyses proposed above represent a comprehensive characterization of changes to cancer progression induced by aberrant PRC2 epigenetic regulation.


\subsubsection*{Pitfalls and alternative approaches}

As PRC2 is a protein complex with multiple subtypes thought to modify its recruitment and enzymatic function \cite{Wassef2017, Holoch2017, Kasinath2018, Laugesen2019a}, we expect different mutations of the subunits of PRC2 to have different impacts.
Therefore, a careful survey of the RNA expression, copy number, and missense mutations to the genes comprising PRC2 will be conducted prior to the above analyses.
However, one of the reasons for studying this phenomenon in COAD and LUAD is because the most common alteration to PRC2 is gain-of-function via the overexpression of \emph{EZH2} \cite{Comet2016MaintainingCancer.}.
Other cancers have frequent loss-of-function through the deletion of any of the PRC2 core subunits, making the analysis far more complex in those systems \cite{Comet2016MaintainingCancer.}.



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Aim 1.2
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection*{Aim 1.2. Determine genetic dependencies caused by aberrant PRC2 function.}


\subsubsection*{Approach}

Differential genetic dependency in PRC2-mutant cell lines.


\subsubsection*{Pitfalls and alternative approaches}


